readme.md

```diff
@@ -154,8 +154,9 @@ programming languages with appropriate bindings for Ruby.
   Deep learning framework for Ruby.
 - [tensorflow](https://github.com/somaticio/tensorflow.rb) - Ruby bindings for [TensorFlow](https://www.tensorflow.org/).
 - [ruby-dnn](https://github.com/unagiootoro/ruby-dnn) - Simple deep learning for Ruby.
-- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch)
+- [torch-rb](https://github.com/ankane/torch-rb) - Ruby bindings for [LibTorch](https://github.com/pytorch/pytorch).
```

Comment:
Why do we need a dot here in the middle of a sentence?

Reply:
All other entries seem to have a full stop, eg lines 155, 154 etc. It seemed that line 157 is missing one.